comptoir-ejb/src/main/java/be/valuya/comptoir/service/SaleService.java

```diff
@@ -346,6 +346,16 @@ public void cancelOpenSale(Sale sale) {
         for (ItemVariantSale itemSale : itemSaleList) {
             removeItemSale(itemSale);
         }
+        AccountingTransaction accountingTransaction = sale.getAccountingTransaction();
+        Company company = sale.getCompany();
+        AccountingEntrySearch accountingEntrySearch = new AccountingEntrySearch();
+        accountingEntrySearch.setAccountingTransaction(accountingTransaction);
+        accountingEntrySearch.setCompany(company);
+        List<AccountingEntry> accountingEntries = accountService.findAccountingEntries(accountingEntrySearch);
+        for (AccountingEntry accountingEntry : accountingEntries) {
+            accountService.removeAccountingEntry(accountingEntry);
```

Comment:
or this (just because we can) instead of the List/for loop:

accountingEntries.forEach(accountService::removeAccountingEntry);